### PR TITLE
python 3.8 compatibility

### LINKED ISSUE #846: Python 3.8 syntax error
I get the following error (with python 3.8)
SyntaxWarning: "is" with a literal. Did you mean "=="?

helper.py:119

if k is 'authorization':

This should use == instead of "is".

### PR DESCRIPTION
**Pull request by ktarplee**
Fixed an issue where == should be used instead of "is" for python 3.8

closes #846

### CODE REVIEW

**Comment by harshavardhana** [comment]
Please fix the title typo @ktarplee `compatibility`

File: minio/helpers.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def dump_http(method, url, request_headers, response, output_stream):
                                                     http_path))
 
     for k, v in list(request_headers.items()):
-        if k is 'authorization':
+        if k == 'authorization':
             # Redact signature header value from trace logs.
             v = re.sub(r'Signature=([[0-9a-f]+)', 'Signature=*REDACTED*', v)
```
Comment (serhiy-storchaka) [inline minio/helpers.py]: It looks suspicious. Is not one of `[` redundant?

Also, `(` and `)` are not needed.
```suggestion
            v = re.sub(r'Signature=[0-9a-f]+', 'Signature=*REDACTED*', v)
```

Comment (harshavardhana) [reply minio/helpers.py]: @serhiy-storchaka this PR doesn't need to address this. Feel free to send PR yourself.

**Comment by harshavardhana** [comment]
@ktarplee can you also update `.github/workflows/pythonpackage-linux.yml` to enable building with python 3.8 ?

**Pull request merged**